src/WebManager/Azure.ContainerRegistry/MediaTypeHelper.cs

```diff
@@ -0,0 +1,22 @@
+namespace Azure.ContainerRegistry
+{
+    static class MediaTypeHelper
+    {
+        static string _acceptMediaTypes;
```

Comment:
I wonder if `GetManifestAcceptMediaTypes()` could be made simpler by  making _acceptMediaTypes a Property...

```suggestion
        static string _acceptMediaTypes => string.Join(';',
                    OCI.MediaTypes.IMAGE_INDEX,
                    OCI.MediaTypes.IMAGE_MANIFEST,
                    Docker.MediaTypes.IMAGE_INDEX,
                    Docker.MediaTypes.IMAGE_MANIFEST
                );
        public static string GetManifestAcceptMediaTypes()
        {
            return _acceptMediaTypes;
        }                      
```

or 

```
        public static string GetManifestAcceptMediaTypes() =>  string.Join(';',
                    OCI.MediaTypes.IMAGE_INDEX,
                    OCI.MediaTypes.IMAGE_MANIFEST,
                    Docker.MediaTypes.IMAGE_INDEX,
                    Docker.MediaTypes.IMAGE_MANIFEST
                );
```

See[ expression-bodied members](https://docs.microsoft.com/en-us/dotnet/csharp/programming-guide/statements-expressions-operators/expression-bodied-members).

Reply:
This could also be a public property itself instead of a method...

Reply:
I don't think expression bodied members are able to cache the value.

Reply:
Fixed in some form.